content/articles/2019-09-tidymodels.Rmarkdown

```diff
@@ -0,0 +1,140 @@
+---
+title: tidymodels updates
+slug: tidymodels-2019-09
+author: Max Kuhn, Edgar Ruiz, and Davis Vaughan 
+description: The latest updates to the tidymodels packages
+date: '2019-09-03'
+categories: [package]
+tags:
+  - tidymodels
+photo:
+  url: https://unsplash.com/photos/rqIfy1UyIzE
+  author: Branden Harvey
+---
+
+We've sent a few packages to CRAN recently. Here's a recap of the changes (and some notes at the bottom): 
+
+## recipes 0.1.6
+ 
+### Breaking Changes
+
+ * Since 2018, a warning has been issued when the wrong argument was used in `bake(recipe, newdata)`. The depredation period is over and `new_data` is officially required.  
```

Comment:
> depredation

deprecation

Reply:
some might argue both :-)

Reply:
I had no idea that was a word but now I'm dying 🤣